openedx/features/content_type_gating/partitions.py

```diff
@@ -147,6 +150,21 @@ def get_group_for_user(cls, course_key, user, user_partition, **kwargs):  # pyli
         if not CONTENT_TYPE_GATING_FLAG.is_enabled():
             return cls.FULL_ACCESS
 
+        # TODO: clean up as part of REV-100
```

Comment:
@cpennington is this the right place for this code to go?

Reply:
Yes.